src/status_im/ui/screens/profile/user/views.cljs

```diff
@@ -25,84 +29,109 @@
+          #(reset! to-hide false))))}
+    ;; TODO - change this to better
+    ;; This is really weird!
+    ;; We show this component whether or not @show is true
+    ;; Why use it at all?
+    ;; because we HAVE to deref `show` in render method for this to work
+    (when true
+      @show?
+      [react/animated-view {:style {:flex           1
+                                    :flex-direction :row
+                                    :margin-left    -40
+                                    :align-items    :center
+                                    :align-self     :stretch
+                                    :opacity        anim-opacity}}
+       [photos/photo photo-path {:size 40}]
+       [react/text {:style {:typography  :title-bold
+                            :line-height 21
+                            :margin-left 16
+                            :margin-right 40
+                            :text-align  :left}}
+        (multiaccounts/displayed-name contact)]])))
 
-(defn qr-viewer-toolbar [label value]
-  [toolbar/toolbar nil
-   (toolbar/nav-button
-    (toolbar.actions/close toolbar.actions/default-handler))
-   [toolbar/content-title label]])
-
-(defn qr-code-share-button [value]
-  (let [link (universal-links/generate-link :user :external value)]
-    [button/button-with-icon
-     {:on-press            #(list-selection/open-share {:message link})
-      :label               (i18n/label :t/share-link)
-      :icon                :main-icons/share
-      :accessibility-label :share-my-contact-code-button
-      :style               styles/share-link-button}]))
-
-(defview qr-viewer []
-  (letsubs [{:keys [value contact]} [:qr-modal]
-            ttt-enabled? [:tribute-to-talk/enabled?]]
-    [react/view styles/qr-code-viewer
-     [status-bar/status-bar {:type :modal-white}]
-     [qr-viewer-toolbar (multiaccounts/displayed-name contact) value]
-     [qr-code-viewer/qr-code-viewer
-      (merge
-       {:style         styles/qr-code
-        :value         value
-        :hint          (i18n/label :t/qr-code-public-key-hint)
-        :legend        (str value)
-        :show-tribute-to-talk-warning? ttt-enabled?}
-       (when-not platform/desktop?
-         {:footer-button qr-code-share-button}))]]))
-
-(defn- show-qr [contact source value]
-  #(re-frame/dispatch [:navigate-to :profile-qr-viewer {:contact contact
-                                                        :source  source
-                                                        :value   value}]))
+(defn my-profile-toolbar [{:keys [public-key photo-path] :as multiaccount} show?]
+  [toolbar/toolbar
+   {:transparent? true
+    :style        {:z-index 100
+                   :elevation 11}}
+   nil
+   [my-profile-toolbar-content multiaccount show?]
+   [toolbar/actions
+    [{:icon      :main-icons/share
+      :icon-opts {:width  24
+                  :height 24}
+      :handler   #(do
+                    (println "show-popover dispatched")
```

Comment:
and also please clean all `println` ,thanks )

Reply:
done!